gradle.properties

```diff
@@ -11,4 +11,18 @@ forge_version=40.2.17
 ftbchunks_version=forge-1802.3.19-build.362
 ftbteams_version=forge-1802.2.11-build.107
 ftblib_version=forge-1802.3.11-build.177
-architectury_version=4.12.94-forge
\ No newline at end of file
+architectury_version=4.12.94-forge
```

Comment:
do we still need this version line?
if we're using file IDs the versions can go, right?